src/unix/linux_like/linux/mod.rs

```diff
@@ -5599,6 +5599,13 @@ extern "C" {
         timeout: ::c_int,
         sigmask: *const ::sigset_t,
     ) -> ::c_int;
+    pub fn epoll_pwait2(
```

Comment:
I ve seen no linux libc exposing it but glibc since beginning of 2022.I think it is safe to put this portion in `linux/gnu/mod.rs` instead (and the semver part). You might still need to mind the CI  outcome somehow due to old linux releases.

Reply:
Thank you! This indeed solves failures.